app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for person in people:
+        person = Person(person["name"], person["age"])
+        person_list.append(person)
+
+    for one_person in people:
+        person = Person.people[one_person["name"]]
+        if one_person.get("wife") is not None:
+            person.wife = Person.people[one_person["wife"]]
+            person.wife.husband = person
```

Comment:
Due to task requirements the code must be:
```suggestion
        if one_person.get("wife") is not None:
            person.wife = Person.people[one_person["wife"]]
        if one_person.get("husband") is not None:
            person.husband = Person.people[one_person["husband"]]
```